device-types/ADVA/FSP-150-CM.yaml

```diff
@@ -0,0 +1,63 @@
+---
+manufacturer: ADVA
+model: FSP-150-CM
+slug: adva-fsp-150-cm
+part_number: '1044004001'
+u_height: 4
+is_full_depth: false
+console-ports:
+  - name: rs-232
+    type: rj-45
+power-ports:
```

Comment:
Are the power supplies on these modular? If so, then the PSUs should be created under `module-bays`, and the associated PSUs should also be created in the module-types folder if they do not already exist

Reply:
Done. Again, I can't find a datasheet for any of this so I've given it a sensible name and model